app/assets/images/payment_icons/bancobrasil.svg

```diff
@@ -0,0 +1,9 @@
+<svg
+    xmlns="http://www.w3.org/2000/svg" width="38" height="24" role="img" viewBox="0 0 38 24" aria-labelledby="pi-bancobrasil">
+    <title id="pi-bancobrasil">Banco Do Brasil</title>
+    <style>.st2{fill-rule:evenodd;clip-rule:evenodd;fill:#263675}</style>
```

Comment:
This comment applies on all other icons:

For the `class` attributes, we want all `<styles>` to be converted into inline attribute. What it means is the following code:

```
<style type="text/css">
  .st1{fill:#FFFFFF;}
</style>
<path xmlns="http://www.w3.org/2000/svg" class="st1" d="M35,1c1.1,0,2,0.9,2,2v18c0,1.1-0.9,2-2,2H3c-1.1,0-2-0.9-2-2V3c0-1.1,0.9-2,2-2H35"/>
```

will become

```
<path xmlns="http://www.w3.org/2000/svg" fill="FFFFFF" d="M35,1c1.1,0,2,0.9,2,2v18c0,1.1-0.9,2-2,2H3c-1.1,0-2-0.9-2-2V3c0-1.1,0.9-2,2-2H35"/>
```